src/main/java/LoginTest.java

```diff
@@ -0,0 +1,31 @@
+import by.issoft.mandrik.steps.LoginSteps;
+import org.testng.Assert;
+import org.testng.annotations.*;
```

Comment:
Avoid asterisks in imports, you can setup IDEA for that

Reply:
ok, thanks!